src/preferences/dialog/dlgprefwaveform.cpp

```diff
@@ -49,6 +49,11 @@ DlgPrefWaveform::DlgPrefWaveform(
         defaultZoomComboBox->addItem(QString::number(100 / static_cast<double>(i), 'f', 1) + " %");
     }
 
+    m_pOverviewMinuteMarkersControl = std::make_unique<ControlObject>(
+            ConfigKey(QStringLiteral("[Waveform]"),
+                    QStringLiteral("DrawOverviewMinuteMarkers")));
```

Comment:
I've noticed a bit late that this PR is introducing new settings using `PascalCase` instead of `snake_case`. Is that something that we can fix before the next release?

Reply:
I don't know if we settled on a convention, but if we want choose one consistently, we should (and can) do it before a release. Though we can only changes keys that have not yet ended up in settings of stable releases.